packages/stylelint-plugin/tests/config-e2e.test.js

```diff
@@ -15,16 +15,23 @@ describe('stylelint-plugin E2E Tests', () => {
     const result = runStylelint('value-keyword-case.*.scss');
 
     const expectedResult = `
+Deprecation warnings:
+ - The "scss/at-import-no-partial-leading-underscore" rule is deprecated.
```

Comment:
We should fix these dependency warnings by replacing our mention of `scss/at-import-no-partial-leading-underscore` with `scss/load-no-partial-leading-underscore`- https://github.com/Shopify/web-configs/blob/8ff1a906437cdcc6ab33e2eb173371ff769fcc8c/packages/stylelint-plugin/index.js#L479

Reply:
https://github.com/Shopify/web-configs/pull/428